aten/src/ATen/cpu/vec/vec256/vec256_float_neon.h

```diff
@@ -479,6 +479,11 @@ template <> class Vectorized<float> {
         vsqrtq_f32(values.val[1]));
   }
   Vectorized<float> reciprocal() const {
+#ifndef C10_MOBILE
```

Comment:
@kimishpatel do you know if reciprocal is at all used on mobile? And shouldn't we always prefer IEEE_754 compliant implementations rather than faster, but less accurate ones?

Reply:
@malfet, I imagine reciprocal might be used somewhere although not sure. I think we should replicate the behavior across platform until we determine it is a perf bottleneck. So lets do that for now. We have some models we do perf tracking across so we can check if this change makes a difference there.